components/concept/toolbar/src/main/java/mozilla/components/definition/toolbar/ToolbarConcept.kt

```diff
@@ -0,0 +1,15 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.definition.toolbar
```

Comment:
We introduce .definition. here (in addition to concept) which concept-engine doesn't have. My personal preference would be to keep the namespace simple i.e. stick to mozilla.components.toolbar and just Toolbar as the classname. Basically, the same as you did with Engine.